sanssouci/lambda_calibration.py

```diff
@@ -118,6 +165,8 @@ def get_pivotal_stats(p0, t_inv=t_inv_linear, K=-1):
         confidence bounds on false positives using reference families.
         Annals of Statistics, 48(3), 1281-1303.
     """
+    # Sort permuted p-values
+    p0 = np.sort(p0, axis=1)
 
     # Step 3: apply template function
     tkInv_all = t_inv(p0)
```

Comment:
```suggestion
    tk_inv_all = t_inv(p0)
``` use underscores for variables, modules and functions, CamelCase for classes, and never mix them !